src/back/jobs/UpdatesMissingForumPost.ts

```diff
@@ -0,0 +1,11 @@
+import UpdateModel from '../../entities/Updates/model'
+import { DiscourseService } from '../../services/DiscourseService'
+import { ProposalService } from '../../services/ProposalService'
+
+export async function restoreMissingUpdatesForumPost() {
+  const affectedUpdates = await UpdateModel.getUpdatesWithoutForumPost()
+  for (const update of affectedUpdates) {
+    const { title } = await ProposalService.getProposal(update.proposal_id)
+    await DiscourseService.createUpdate(update, title)
```

Comment:
It'd be good to have some feedback (return value/log) so that we know all affected updates had their forum post created 👌